### PR TITLE
cleanup statistics aggregators

### PR DESCRIPTION
**Pull request by jakebiesinger**
@anbangx @JavierJia @Elmira88 @Nan-Zhang 
All statistics can now be handled in the compute functions hadoop Counters, like so:
- `getCounters().findCounter(GraphMutations.Num_RemovedBridges).increment(1);`
- or equivalently but slower: `getCounters().findCounter("GraphMutations", "Num_RemovedBridges").increment(1);`
- or any group and counter name: `getCounters().findCounter("genomix", "coolness-factor").increment(Long.MAX_VALUE);`

### CODE REVIEW

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/tipremove/TipRemoveVertex.java [hunk 1]
```diff
@@ -18,10 +17,16 @@
 /**
  * Remove tip or single node when kmerLength < MIN_LENGTH_TO_KEEP
  * Details: Sequencing errors at the ends of the reads form "tips": short, low coverage nodes
+<<<<<<< Updated upstream
  * 			with in-degree + out-degree = 1 (they either have a single edge in or a single edge out).
  * 			The algorithm identifies these nodes and prunes them from the graph. This is then followed
  * 			by recompressing the graph.
  * 
+=======
```
Comment (JavierJia) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/tipremove/TipRemoveVertex.java]: Comment conflict ?

Comment (jakebiesinger) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/tipremove/TipRemoveVertex.java]: Good catch.  Fixed.

**Comment by jakebiesinger** [comment]
May I merge this in?

**Comment by anbangx** [comment]
I don't have time to check this. But you can merge first, I will check it
later.

On Mon, Dec 2, 2013 at 12:15 PM, Jake Biesinger notifications@github.comwrote:

> May I merge this in?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/uci-cbcl/genomix/pull/76#issuecomment-29653371
> .

## 

Best Regards,

Anbang Xu

**Comment by JavierJia** [comment]
:+1:

**Pull request merged**